src/Entities/Blog.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+readonly class Blog {
+    public int $id;
+    public string $title;
+    public string $content;
+    public string $extract;
+    public int $authorId;
+    public string $postTime;
+
+    public function __construct(int $id, string $title, string $content, int $authorId, string $postTime)
+    {
+        $this->id = $id;
+        $this->title = $title;
+        $this->extract = substr($content, 0, 100) . '...';
+        $this->content = $content;
+        $this->authorId = $authorId;
+        $this->postTime = gmdate("d/m/y", strtotime($postTime));
+    }
```

Comment:
You should write a couple of unit tests for this now, now that there is some actual stuff happening here.

I think two tests:
1) Instantiate a Blog, and assert that the extract is correct
2) Instantiate a blog and asser that the postTime is formatted correctly (you can do this by passing in a date string stolen from the DB, like '2024-03-04 10:23:52'

Reply:
Thanks! Tests added now :)